src/lazy/text/buffer.rs

```diff
@@ -3255,25 +3255,25 @@ mod tests {
             "(:foo (1 2 3))",
             "(:foo \"foo\")",
             "(:foo foo)",
-            "(:4)",
```

Comment:
🗺️ Another unit test whose macro ID needed updating. This one is trickier to update automatically because we'd need to do string formatting for each test to construct an updated expression.